projects/atft/src/lib/raycaster/raycaster-group.directive.ts

```diff
@@ -6,7 +6,7 @@ import * as THREE from 'three';
 
 export interface RaycasterEmitEvent {
   component: AbstractObject3D<any>;
-  face?: THREE.Face;
+  face?: THREE.Face3;
```

Comment:
Isn't removed in r126 from core? 
https://github.com/mrdoob/three.js/releases/tag/r126